lib/open_food_network/distribution_change_validator.rb

```diff
@@ -9,8 +9,10 @@ def can_change_to_distribution?(distributor, order_cycle)
   end
 
   def variants_available_for_distribution(distributor, order_cycle)
-    product_distribution_variants = distributor.andand.product_distribution_variants || []
-    order_cycle_variants = order_cycle.andand.variants_distributed_by(distributor) || []
+    product_distribution_variants = []
```

Comment:
I actually prefer the old version. Double-assignments are bad style.

Reply:
I'll keep the new version. I dont like andand, I think it's bad in terms of readability, in the first version it's not clear that the result is [] if distributor is nil, in the new version, it's explicit.
I like double assignments in cases like this, they make code simple, forcing everything into single assignments makes the code unnecessarily complicated. this is avoiding using the readers brain capacity when it is not necessary.
but it's a personal preference...

Reply:
I don't have a preference for double assignments or `||` but I do not like `andand` at all. To me, it feels like a symptom of unconfident programming which tends to lead to bloat.